### DISCUSSION TITLE
3 or more bots chatting with each other

### DISCUSSION THREAD

**Discussion by juane2010** [facebookresearch/ParlAI#3450] [General]
i saw the self_chat script and i was impressed on how realistic the conversation was. Is it possible to create one for multiples bot chatting among them with the current release?
Thank you

**Comment by stephenroller** [comment]
Hi there, thanks for asking on discussions.

Unfortunately our systems aren't very well set up for multi-party chat right now. You can still _manually_ implement it by creating a custom World and manually having multiple bots observe, but it's a lot of work, and only works at deployment time.

**Reply by juane2010** [reply]
thank you for the response. Is this feature planned in future releases?

**Reply by stephenroller** [reply]
It's not on our roadmap through June 2021.

**Reply by juane2010** [reply]
ok thank you i think it would be a interesting to see how a communication evolve in such environment.
